CONTRIBUTING.md

```diff
@@ -6,7 +6,24 @@ We really appreciate and value contributions to OpenZeppelin Contracts for Cairo
 
 Before starting development, please [create an issue](https://github.com/OpenZeppelin/cairo-contracts/issues/new/choose) to open the discussion, validate that the PR is wanted, and coordinate overall implementation details.
 
-Also, consider that snake case is used for Cairo development in general due to its strong Python bias, but in order to be compliant with the standards, EIP implementations must use camelCase. Therefore, EIP-based contracts will have their external functions written in camelCase while the rest of the codebase will be in snake_case.
+Also, consider that snake case is used for Cairo development in general due to its strong Python bias.
+This project's opinionated approach consists of camelCasing all exposed function names and their parameters:
```

Comment:
```suggestion
This project follows our [Extensibility pattern](https://docs.openzeppelin.com/contracts-cairo/extensibility), camelCasing all exposed function names and their parameters:
```

given how contentious the topic is, i'd avoid saying this is opinionated (although it is, but no need to raise it). also the extensibility link early in the section should help shed some light on what "internal" means, otherwise it goes unmentioned